common/common-ui/src/main/java/com/duckduckgo/common/ui/Theming.kt

```diff
@@ -89,6 +74,8 @@ fun AppCompatActivity.getThemeId(theme: DuckDuckGoTheme): Int {
     return when (theme) {
         SYSTEM_DEFAULT -> getSystemDefaultTheme()
```

Comment:
`getSystemDefaultTheme` also needs to be updated to return either the regular or experiment themes.

Reply:
During internal testing we only want to change Light / Dark. Users on system theming won’t be able to see the experimental colors.

Reply:
This sounds somewhat counter-intuitive for me. The new flag governs whether the experimental light/dark theme is applied, we shouldn't require from internal users any additional implicit steps. The default setting is the "system theme", and in the end it applies one of the light or dark themes, so in my opinion it should respect the experiment flag as well.

Reply:
"we shouldn't require from internal users any additional implicit steps”. this is still to be decided, for now we will keep it simple and iterate in the project.